config/software/protobuf-py.rb

```diff
@@ -0,0 +1,49 @@
+name "protobuf-py"
+default_version "3.1.0"
+
+dependency "python"
+dependency "setuptools"
+dependency "pip"
+dependency "six"
+
+source :url => "https://github.com/google/protobuf/releases/download/v#{version}/protobuf-python-#{version}.tar.gz",
+       :sha256 => "0bc10bfd00a9614fae58c86c21fbcf339790e48accf6d45f098034de985f5405"
+
+relative_path "protobuf-#{version}/python"
+
+env = {}
+
+if ohai['platform_family'] == 'mac_os_x'
+    env['MACOSX_DEPLOYMENT_TARGET'] = '10.9'
+end
+
+build do
+    ship_license "https://raw.githubusercontent.com/google/protobuf/3.1.x/LICENSE"
+
+    build_options = ""
+    pip_build_options = ""
+
+    # C++ runtime
+    if ohai['platform_family'] != 'rhel'
```

Comment:
Could you add a comment here to explain why we don't build the cpp extensions? (for future reference)